docs/sphinx/examples/python/operators.ipynb

```diff
@@ -52,7 +52,7 @@
     "\n",
     "The `pauli_word` type specifies a string of Pauli operations (e.g. ‘XYXZ’) and is convenient for applying operations based on exponentiated Pauli words.  The code below demonstrates how a list of Pauli words, along with their coefficients, are provided as kernel inputs and converted into operators by the `exp_pauli` function.\n",
     "\n",
-    "The cell below applies the following operation: $e^{i(0.432XZY +0.324IXX)}$"
+    "The cell below applies the following operation: $e^{i(0.432XYZ)}.e^{i(0.324IXX)}$"
```

Comment:
I think we need to swap the order: with the list `words = ['XYZ', 'IXX']`, `XYZ` will be applied first, aka, `e^{i(0.432XYZ)}` will hit the state vector first.